disease/version.py

```diff
@@ -1,2 +1,2 @@
 """Disease normalizer version"""
-__version__ = "0.3.0"
+__version__ = "0.2.16"
```

Comment:
I think it should be 0.3.1 if off of staging branch
```suggestion
__version__ = "0.3.1"
```

Reply:
Oh this must've been an automerge thing. No need to issue a new version yet